frontend/coprs_frontend/coprs/views/apiv3_ns/apiv3_permissions.py

```diff
@@ -11,6 +11,20 @@
 from . import GET, PUT, editable_copr, get_copr
 
 
+@apiv3_ns.route("/project/permissions/can_build_in/<ownername>/<projectname>")
+@api_login_required
+def can_build_in(ownername, projectname):
+    copr = get_copr(ownername, projectname)
+    if not flask.g.user.can_build_in(copr):
+        raise BadRequest("User '{0}' is not allowed to build in '{1}'"
+                         .format(flask.g.user.name, copr.full_name))
+
+    # I don't really know what to return. Perhaps we can start with an empty
+    # dict and add information as we need it? For now, it's important that it
+    # returns 200 status code.
```

Comment:
The call has "assert"-like, which is a bit uncommon in APIs.  We also return BadRequest (error 400).
I'm curious if there's some "guidance" in some API best-practices documentation...?